src/protocolsupport/protocol/typeremapper/tile/TileEntityRemapper.java

```diff
@@ -309,7 +310,20 @@ public void accept(TileEntity tile) {
 			},
 			ProtocolVersionsHelper.ALL_PE
 		);
-
+		register(
```

Comment:
The tag selection should be inside TileEntitySkullRemapper.
This way just adds unnesessary overhead.